src/Pollers/DeviceMappingPoller.php

```diff
@@ -161,6 +161,7 @@ private function getDeviceMapper(array $hostWithDeviceType, Device $device)
                 break;
             case "1.3.6.1.4.1.17713.21":
             case "1.3.6.1.4.1.17713.21.1.1.2":
+            case "1.3.6.1.4.1.17713.21.9.37" || "1.3.6.1.4.1.17713.21.9.40" || "1.3.6.1.4.1.17713.21.9.43" || "1.3.6.1.4.1.17713.21.9.44":
```

Comment:
Sorry I took forever to look at this, I'll merge it quickly, but this should be:

`case "1.3.6.1.4.1.17713.21.9.37":`
`case "1.3.6.1.4.1.17713.21.9.40":`

etc